master/internal/api_experiment.go

```diff
@@ -1489,15 +1489,29 @@ func (a *apiServer) ContinueExperiment(
 	if err != nil {
 		return nil, fmt.Errorf("getting experiment trials: %w", err)
 	}
-	if len(trialsResp.Trials) != 1 {
-		return nil, status.Error(codes.FailedPrecondition, fmt.Sprintf(
-			"experiment needs exactly one trial got %d instead", len(trialsResp.Trials)))
-	}
-	trialID := int(trialsResp.Trials[0].Id)
+	var configBytes []byte
+	if req.IsSingle {
+		if len(trialsResp.Trials) != 1 {
+			return nil, status.Error(codes.FailedPrecondition, fmt.Sprintf(
+				"experiment needs exactly one trial got %d instead", len(trialsResp.Trials)))
+		}
+		//
+		configBytes, err = a.parseAndMergeContinueConfig(int(req.Id), req.OverrideConfig)
+		if err != nil {
+			return nil, err
+		}
+	} else {
+		activeConfig, err := a.m.db.ActiveExperimentConfig(int(req.Id))
+		if err != nil {
+			return nil, fmt.Errorf("loading active config for experiment %d: %w", req.Id, err)
+		}
 
-	configBytes, err := a.parseAndMergeContinueConfig(int(req.Id), req.OverrideConfig)
```

Comment:
This change is duplicating logic a little, we have multiple places where we load config

```parseAndMergeContinueConfig``` should just return the original config ```if req.OverrideConfig ==  ""```

maybe instead we can just change this if statement to error if OverrideConfig is set and it is not a searcher?

https://github.com/determined-ai/determined/blob/95f2ad797707988fd947325ea6e008aeb5546029/master/internal/api_experiment.go#L1451

Reply:
done